lib/config/presets/internal/default.ts

```diff
@@ -456,6 +460,19 @@ export const presets: Record<string, Preset> = {
       'Apply labels <code>{{arg0}}</code> and <code>{{arg1}}</code> to PRs',
     labels: ['{{arg0}}', '{{arg1}}'],
   },
+  labelUpdateTypes: {
+    description:
+      'Apply label per update type, prefixed by <code>{{arg0}}</code> and suffixed by <code>{{arg1}}</code> to PRs',
+    packageRules: options
+      .find((it) => it.name === 'matchUpdateTypes')
+      .allowedValues.map(
+        (t) =>
```

Comment:
```suggestion
        (t: UpdateType) =>
```

Reply:
That doesn't work, as the `.allowedValues` is of type `string[]`. I can map to `UpdateType` prior to mapping to a `PackageRule`.